packages/components/inputs/select-utils/src/clear-indicator/clear-indicator.tsx

```diff
@@ -37,6 +37,10 @@ const ClearIndicator = (props: TClearIndicator) => {
       style={getStyles('clearIndicator', props) as CSSProperties}
       title={intl.formatMessage(messages.clearButtonLabel)}
       aria-label={intl.formatMessage(messages.clearButtonLabel)}
+      // overriding the default `aria-hidden` prop value to make the component accessible by keyboard - https://github.com/JedWatson/react-select/issues/4793
+      aria-hidden={false}
```

Comment:
`react-select` allows using customized components passed as `components` prop. `ClearIndicator` is one of the supported customizable components and our implementation of `SelectInput` uses a customized `ClearIndicator`.

The issue reported by Lighthouse is caused by the default props passed to the `ClearIndicator` component (namely `aria-hidden: true`). This is the [rationale](https://github.com/JedWatson/react-select/issues/4793) of the lib creators. Please mind though that they keep the `ClearIndicator` not accessible by keyboard by default. In the `react-select` [docs examples](https://react-select.com/components#adjusting-the-styling) in each of the cases the container element has attribute `aria-hidden=“true”` and the svg element has attributes: `aria-hidden:”true”` and `focusable="false”`.

 I am assuming this is not the way we wish that to work (as we use a button for the `ClearIndicator` component).

I have followed the advice from the GH issue to override the default bahavior.

Reply:
So it seems we have two options:
* Do not hide the reset button to screen readers
* Do explicitly hide inner components of `ClearIndicator` (the button) to screen readers

It seems they defaulted to the second one in the library to mimic the behaviour of reset handle in HTML search inputs.

From my point of view, I prefer the option you've taken here 👍 .